src/server/index.js

```diff
@@ -143,7 +144,11 @@ async function start() {
 	// this middleware is never called in this case
 	// eslint-disable-next-line no-unused-vars
 	app.use((err, req, res, next) => {
-		if (err.error.message.startsWith("Cast to ObjectId failed for value")) {
+		if (
+			err &&
+			err.error &&
+			err.error.message.startsWith("Cast to ObjectId failed for value")
```

Comment:
Maybe we can use some new syntax for that and keep it short:
```suggestion
			err?.error?.message.startsWith("Cast to ObjectId failed for value")
```
This syntax was developed for exactly this use-case.

I think we then need this bable plugin:
https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining

Reply:
Wow awesome!! I was waiting for this feature since angularjs had it. Of course let’s use it. We can also use it in the whole project!!! Awesomely:)

Reply:
I tried to add it but it's not working out of the box.

![image](https://user-images.githubusercontent.com/3246782/62124461-dc5b8980-b2fc-11e9-8e92-5d24125d91bc.png)


Lets better add it in another PR